java/java.source.base/test/unit/src/org/netbeans/modules/java/source/indexing/VanillaCompileWorkerTest.java

```diff
@@ -1958,13 +1958,7 @@ protected void tearDown() throws Exception {
     }
 
     public static Test suite() {
-        if (NoJavacHelper.hasNbJavac()) {
-            return new VanillaCompileWorkerTest("noop");
-        } else {
-//            return new VanillaCompileWorkerTest("testAnonymousClasses");
-//            return new VanillaCompileWorkerTest("testPreserveValidInitializers");
-            return new NbTestSuite(VanillaCompileWorkerTest.class);
-        }
+        return new VanillaCompileWorkerTest("noop");
```

Comment:
Sorry, but the other branch should be preserved:
```
return new NbTestSuite(VanillaCompileWorkerTest.class);
```

Or simply remove the `suite` method, as this should be the default.

The intent here was to not run the test with the old nb-javac, but the new one is basiclly a javac from the JDK, so the test should run.

Reply:
done (it confused me a bit since both javac versions are now similar, so there would be plausible reasons to run vanilla tests but also nb-javac tests :))